_data/chains/eip155-314.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Filecoin — Mainnet",
+  "chain": "FIL",
+  "status": "incubating",
+  "rpc": [],
```

Comment:
For rpc we can add:
```
"rpc" [
     "https://api.node.glif.io/rpc/v0"
].
```

Reply:
Sure, I was under the impression that these RPCs needed to follow the Ethereum JSON-RPC interface, hence why I left them blank while the that implementation is underway. Happy to add these.

Reply:
Yes. Can leave it blank for now, this is for EVM flavoured chains only, while the rest of CAIP is setting the standards on a per chain basis for things like wallet clients that want to talk across them.